build-source/snp.index

```diff
@@ -140,7 +141,6 @@ france4hd=france4
 france5hd=france5
 euronewsdeutsch=euronews
 realityxtra2=realityxtra2
-classichits=classichits
```

Comment:
Please re-add this link.
Unless  service exists on your box it wont show. When it reverts back it will show automatically.